models/DataObject/ClassDefinition.php

```diff
@@ -504,7 +504,7 @@ private function generateClassFiles($generateDefinitionFile = true)
                             $this->getName()
                         ).' getBy'.ucfirst(
                             $def->getName()
-                        ).'($field, $value, $locale = null, $limit = 0, $offset = 0)'."\n";
+                        ).'($field, $value, $locale = null, $limit = 0, $offset = 0, $objectTypes = null)'."\n";
```

Comment:
since this changes the class generation code. we need a migration, which calls`generateClassFiles` for exisiting generated classes something like https://github.com/pimcore/pimcore/blob/6.9/bundles/CoreBundle/Migrations/Version20200529121630.php#L34